src/modules/logger/logger.h

```diff
@@ -346,13 +344,16 @@ class Logger : public ModuleBase<Logger>
 	uORB::Subscription				_manual_control_sp_sub{ORB_ID(manual_control_setpoint)};
 	uORB::Subscription				_vehicle_command_sub{ORB_ID(vehicle_command)};
 	uORB::Subscription				_vehicle_status_sub{ORB_ID(vehicle_status)};
-	uORB::SubscriptionInterval			_log_message_sub{ORB_ID(log_message), 20};
-
-	param_t						_sdlog_profile_handle{PARAM_INVALID};
-	param_t						_log_utc_offset{PARAM_INVALID};
-	param_t						_log_dirs_max{PARAM_INVALID};
-	param_t						_mission_log{PARAM_INVALID};
-	param_t						_boot_bat_only{PARAM_INVALID};
+	uORB::SubscriptionInterval		_log_message_sub{ORB_ID(log_message), 20};
+
+	DEFINE_PARAMETERS(
+		(ParamInt<px4::params::SDLOG_UTC_OFFSET>) _param_sdlog_utc_offset,
+		(ParamInt<px4::params::SDLOG_DIRS_MAX>) _param_sdlog_dirs_max,
+		(ParamInt<px4::params::SDLOG_PROFILE>) _param_sdlog_profile,
+		(ParamInt<px4::params::SDLOG_MISSION>) _param_sdlog_mission,
+		(ParamInt<px4::params::SDLOG_BOOT_BAT>) _param_sdlog_boot_bat,
+		(ParamInt<px4::params::SDLOG_UUID>) _param_sdlog_uuid
```

Comment:
```suggestion
		(ParamBool<px4::params::SDLOG_UUID>) _param_sdlog_uuid
```

Reply:
@bkueng thanks, Will be fixed. 
I wonder Why it didn't fail on compilation. because using ParamFloat vs ParamInt does fail.